scrape/scrape.py

```diff
@@ -222,34 +200,26 @@ async def write_service(
     service_prefix: str, service_name: str, actions: Set[str]
 ) -> None:
     content: List[str] = []
-    if service_prefix in LEGACY_ARNS:
-        content.append("import warnings")
     content.append(HEADER)
 
-    content.append(f"service_name = '{service_name}'")
-    content.append(f"prefix = '{service_prefix}'")
+    content.append(f'service_name = "{service_name}"')
+    content.append(f'prefix = "{service_prefix}"')
     content.append("")
     content.append("")
     content.append(CLASSES_S3 if service_prefix == "s3" else CLASSES)
     content.append("")
-    if service_prefix in LEGACY_ARNS:
-        content.append(
-            LEGACY_ARN_CLASS.format(
-                lower=service_prefix, upper=service_prefix.upper()
-            )
-        )
-        content.append("")
 
     for action in sorted(actions):
         action = action.strip()
         # Handle action such as "ReEncrypt*"
         if action[-1] == "*":
             action = action[:-1]
-        # Wrap lines for pep8
-        if len(action) > 30:
-            action_string = "{action} = \\\n    Action('{action}')"
+        # Wrap lines for black
+        linelength = len(action) * 2 + 13
+        if linelength >= 88:
+            action_string = '{action} = Action(\n    "{action}"\n)'
```

Comment:
Instead of hard-coding the line length here (it can be changed in black's config), I'd suggest to let black format the code.

Possibly in `.github/workflows/scrape.yml` right before `git add awacs/`.

Reply:
And maybe add a `print("And now run 'black awacs/'", flush=True)` to `if __name__ == "__main__":` as a hint.

Reply:
Good feedback. Fixed.